addons/mail/models/mail_tracking_value.py

```diff
@@ -35,10 +34,21 @@ class MailTracking(models.Model):
 
     tracking_sequence = fields.Integer('Tracking field sequence', readonly=1, default=100, oldname='track_sequence')
 
+    @property
```

Comment:
Not sure using a property is better than using a getter in the context of Odoo

cc: @tde-banana-odoo @Xavier-Do @tivisse